.cruft.json

```diff
@@ -7,7 +7,7 @@
       "name": "TopoLVM",
       "slug": "topolvm",
       "parameter_key": "topolvm",
-      "test_cases": "defaults legacy",
+      "test_cases": "defaults legacy noalertrules",
```

Comment:
Looks like the new test case wasn't added through `commodore component update -t noalertrules /path/to/component`. The new test case is missing in the GitHub actions config.